Src/WindowsServer/WindowsServer.Shared.Tests/FirstChanceExceptionStatisticsTelemetryModuleTest.cs

```diff
@@ -300,6 +300,118 @@ public void FirstChanceExceptionStatisticsTelemetryModuleWillNotDimCapTheSameOpe
+            });
+
+            using (var module = new FirstChanceExceptionStatisticsTelemetryModule())
+            {
+                module.Initialize(this.configuration);
+
+                module.DimCapTimeout = DateTime.UtcNow.Ticks - 1;
+
+                for (int i = 0; i < 200; i++)
+                {
+                    if (i == 101)
+                    {
+                        module.DimCapTimeout = DateTime.UtcNow.Ticks - 1;
+                    }
+
+                    try
+                    {
+                        // FirstChanceExceptionStatisticsTelemetryModule will process this exception
+                        throw new Exception("test");
+                    }
+                    catch (Exception exc)
+                    {
+                        // code to prevent profiler optimizations
+                        Assert.Equal("test", exc.Message);
+                    }
+                }
+            }
+
+            Assert.Equal(200, metrics.Count);
+            Assert.Equal(200, this.items.Count);
+        }
+
+        [TestMethod]
+        public void FirstChanceExceptionStatisticsTelemetryExceptionsAreThrottled()
+        {
+            var metrics = new List<KeyValuePair<Metric, double>>();
+            this.configuration.MetricProcessors.Add(new StubMetricProcessor()
+            {
+                OnTrack = (m, v) =>
+                {
+                    metrics.Add(new KeyValuePair<Metric, double>(m, v));
+                }
+            });
+
+            this.configuration.TelemetryInitializers.Add(new StubTelemetryInitializer()
+            {
+                OnInitialize = (item) =>
+                {
+                    item.Context.Operation.Name = "operationName";
+                }
+            });
+
+            using (var module = new FirstChanceExceptionStatisticsTelemetryModule())
+            {
+                module.Initialize(this.configuration);
+
+                module.TargetMovingAverage = 50;
+
+                for (int i = 0; i < 200; i++)
+                {
+                    try
+                    {
+                        // FirstChanceExceptionStatisticsTelemetryModule will process this exception
+                        throw new Exception("test");
+                    }
+                    catch (Exception exc)
+                    {
+                        // code to prevent profiler optimizations
+                        Assert.Equal("test", exc.Message);
+                    }
+                }
+            }
+
+            int countProcessed = 0;
+            int countThrottled = 0;
+
+            foreach (KeyValuePair<Metric, double> items in metrics)
+            {
+                if (items.Key.Dimensions.Count == 1)
+                {
+                    countThrottled++;
+                }
+                else
+                {
+                    countProcessed++;
+                }
+            }
+
+            Assert.Equal(166, countProcessed);
```

Comment:
I would think that either will suffice. To avoid chancing numbers in the test in the future we can put a formula here. On the other hand, the formula in the test will be testing the identical formula in the code. So, if both have the bug - it might go undetected. From such a perspective, it looks like we can go with the simple comment here.